tests/org.jboss.tools.examples.ui.bot.test/resources/servers/wildfly-blacklist-test-errors-regexes.json

```diff
@@ -27,5 +27,73 @@
 		".*Element 'value' must have no element.*",
 		".*Value 'param1 flow1 value' is not facet-valid with respect to pattern.*",
 		".*Invalid content was found starting with elemen 'flow-call'.*"
+	],
+	"spring-greeter":[
+		".*Can not find the tag library descriptor for \"http://java.sun.com/jsp/jstl/core\".*",
+		".*Missing artifact org.apache.taglibs:taglibs-standard-impl:jar:1.2.6-RC1.*",
+		".*Missing artifact org.apache.taglibs:taglibs-standard-spec:jar:1.2.6-RC1.*",
+		".*Plugin execution not covered by lifecycle configuration: org.apache.maven.plugins:maven-checkstyle-plugin:3.0.0:checkstyle.*",
```

Comment:
I suppose this plugin can be installed via target platform configurator(see https://github.com/zcervink/jbosstools-integration-tests/blob/JBIDE-26558/tests/org.jboss.tools.examples.ui.bot.test/pom.xml#L87) as it has been done with different check-style plugin(https://github.com/zcervink/jbosstools-integration-tests/blob/JBIDE-26558/tests/org.jboss.tools.examples.ui.bot.test/pom.xml#L98).

Reply:
Fixed.